parquet/src/main/java/org/apache/iceberg/parquet/Parquet.java

```diff
@@ -379,37 +384,31 @@ public DeleteWriteBuilder equalityFieldIds(int... fieldIds) {
           appenderBuilder.build(), FileFormat.PARQUET, location, spec, partition, keyMetadata, equalityFieldIds);
     }
 
-
-    public <T> PositionDeleteWriter<T> buildPositionWriter() throws IOException {
+    public <T> PositionDeleteWriter<T> buildPositionWriter()
+        throws IOException {
```

Comment:
Does this line need to change? I'm fine removing the empty line, but I think `throws` can still fit on the previous line.

Reply:
I can revert it.